test/extended/apiserver/apply.go

```diff
@@ -172,3 +181,19 @@ func createResource(
 
 	return resourceClient, &unstructuredObj, testNamespace
 }
+
+// waitForSATokens waits for the kube-controller-manager to populate the SA from the unstructured object
+// to be populated with a secret defining its tokens
+func waitForSATokens(saClient corev1client.ServiceAccountInterface, unstructuredObj map[string]interface{}) {
```

Comment:
odd interface. Would be simpler with just the name as string, and the string extracted at the call site.

Reply:
The function is very specific to this test case, I merely extracted the necessary steps not to pollute the code above.